igor_src/Utils.h

```diff
@@ -667,4 +669,10 @@ void close_progress_bar(std::ostream&, std::string prefix_message = "", size_t p
 uint64_t draw_random_64bits_seed();
 
 
+typedef std::unordered_map<std::string,std::string> UMCodonTable;
```

Comment:
I'm not sure if we'll do anything speed critical with this so it might indeed be more than enough to use an unordered_map, but hash functions are computationally intensive and since here the number of entries (the codon alphabet) is fixed an array inside a struct/class could be faster.
Up to you

Reply:
This isn't gonna make the program run considerably slower, but yes this could be improved later.